news/news.md

```diff
@@ -3,6 +3,19 @@ All issues and bug reports must be opened at [GitHub](https://github.com/pymedus
 
 <br/>
 
+#### 2019-$$-$$ ####
+       ^^ ^^
+## Upcoming new Docker image version
+
+If you're using the [pymedusa/Medusa Docker image](https://hub.docker.com/r/pymedusa/medusa), this message is for you!  
+Talking points: (**_make this nicer_**)
+- We are updating our Docker image to use Python 3.7.
+- By removing some unneeded files and folders, we are able to reduce the size of the image from around 230mb (on latest/master tag) down to around 160mb.
+- Please note that we are also removing the automatic restart functionality from our image.  
+  We suggest using the built-in [restart policy feature](https://docs.docker.com/config/containers/start-containers-automatically/#use-a-restart-policy) when running the container instead.
+
```

Comment:
I think this is good to go. Maybe let's add that we plan to release the new image to develop next week? Also, that besides the restart policy, no action needs to be taken?

Reply:
Im still not sure about the dogpile cache?

Reply:
Shouldn't be an issue since we replace it? https://github.com/pymedusa/Medusa/commit/e769def184eaed5efeb976d8c2423d0d8a61c744#diff-8dae95e92447489197bc2654f723edaeR50

Reply:
@medariox Working on it right now, thanks.

@p0psicles
I actually concluded that that Docker users (ours, and linuxserver's) will not be affected by this issue at all since they both have `gdbm` installed, which is still supported in Python 3 (`dbm.ndbm` IIRC), unlike `dbhash`/`bsddb`, the default for Python 2 if no better framework is available, that was removed in Python 3.

And in any case, as @medariox said, we have the code you set up that removes the `DBM` files when a major Python version change is detected.